Privacy.md

```diff
@@ -0,0 +1,85 @@
+# Fruit Radar Privacy Policy.
+
+This privacy policy document is to clearly state and provides information on what, how and why personal data or information is collected, used and disclosed when this software is utilized and the choices you have associated with that data.
+
+The objective of this policy document is to provide an open environment between developer and users of this software by openly stating what information the developer collects or not. By utilizing this software, it means you accept all the terms and conditions of the developer peculiar to only Fruit Radar software. This Policy applies to this software and the developer would not be held responsible for the content or privacy practices on any third party app not operated by the developer to which this software links or that links to this software, therefore it is advisable to read up the privacy policies of third party services that may be linked to this software. <hr>
+
+## Permission and Access
+
+Permissions and access are simply the device permissions the software requires to function and the access enabling this permission grants.
+To function efficiently and effectively as developed and designed, Fruit Radar will require some device permissions. The permissions are;
+
+- **Camera**
+
+The enablement of the camera permission grants the app access to your device camera to take pictures and videos. 
+
+- **Location**
+
+When the location permission is enabled the software has access to your precise location(GPS and network-based).
+
+- **Storage**
+
+Permission to access device storage allows the software read, modify or delete contents of your SD card. 
+
+The software also has some other capabilities which include;
+
+- Ability to have full network access which means the app is capable of creating network sockets and can use customized network protocols. This capability does not require the software to send data to the internet.
+- Control vibration: This app can control your device vibrator.
+
+The above permissions can be revoked at any time you desire, but revoking these permissions will cause the software to cease to function. These permissions work hand-in-hand simultaneously to ensure smooth operations and utilization of the software. This is privy to only users with devices running Android 6.0 upwards. <hr>
```

Comment:
The software will still function in a way. Just some features are not there. All permissions have a check so that the software should not crash.

Reply:
Oh okay. Should I correct this in the document?

Reply:
If you like, you can. I can do this, too. I created an issue: https://github.com/niccokunzmann/mundraub-android/issues/204
The purpose for the change is the translation. 
Embedding in the app can be done without the change.

Reply:
You are free to add changes any time. I tend to merge fast to give everyone the ability to contribute to the release.

Reply:
I invited you. https://github.com/niccokunzmann/mundraub-android/invitations If you like, you can merge pull requests when you accept the invitation.